ouroboros-consensus-cardano-test/test/Main.hs

```diff
@@ -9,7 +11,7 @@ import qualified Test.Consensus.Cardano.Serialisation (tests)
 import qualified Test.ThreadNet.Cardano (tests)
 
 main :: IO ()
-main = defaultMainWithIohkNightly tests
+main = sodiumInit >> defaultMainWithIohkNightly tests
```

Comment:
For the non-test code: call `sodiumInit` here: https://github.com/input-output-hk/cardano-node/blob/512a82974fb9b5745d1152a7ab0623f02e8fe45e/cardano-cli/src/Cardano/CLI/TopHandler.hs#L69